### DISCUSSION TITLE
Convention of Off-Diagonal Inertia-Tensor-Terms

### DISCUSSION THREAD

**Discussion by MarkusZimmerDLR** [DLR-SL/CPACS#836] [Q&A]
Dear CPACS-Folks,

what kind of convention regarding the inertia tensor off diagonals does CPACS use?

See https://en.wikipedia.org/wiki/Moment_of_inertia#Alternate_inertia_convention.

The latter is used by many CAD and CES software applications.

Unfortunately, I was not able to find a clarification in the documentation by my own.

Thank you,
Markus

**Comment by MarAlder** [comment]
@dlr-bt @sfreund-DLR @sdeinert @rmaierl maybe?
